test/integration/node-middleware.test.ts

```diff
@@ -0,0 +1,90 @@
+import { createServer } from "http";
+
+import fetch from "node-fetch";
+
+import { Webhooks, createNodeMiddleware, sign } from "../../src";
+import { EmitterWebhookEvent } from "../../src/types";
+import { pushEventPayload } from "../fixtures";
+
+const signatureSha256 = sign(
+  { secret: "mySecret", algorithm: "sha256" },
+  JSON.stringify(pushEventPayload)
+);
+
+describe("createNodeMiddleware(webhooks)", () => {
+  it("README example", async () => {
+    const webhooks = new Webhooks<EmitterWebhookEvent>({
+      secret: "mySecret",
+    });
+
+    webhooks.on("push", (event) => {
+      expect(event.id).toBe("123e4567-e89b-12d3-a456-426655440000");
+    });
+
+    const server = createServer(createNodeMiddleware(webhooks)).listen();
+
+    // @ts-expect-error complains about { port } although it's included in returned AddressInfo interface
+    const { port } = server.address();
+
+    const response = await fetch(
+      `http://localhost:${port}/api/github/webhooks`,
+      {
+        method: "POST",
+        headers: {
+          "X-GitHub-Delivery": "123e4567-e89b-12d3-a456-426655440000",
+          "X-GitHub-Event": "push",
+          "X-Hub-Signature-256": signatureSha256,
+        },
+        body: JSON.stringify(pushEventPayload),
+      }
+    );
+
+    expect(response.status).toEqual(200);
```

Comment:
```suggestion
    expect(response.status).toBe(200);
```

nit: prefer using `toBe` for literal matchers (or otherwise `toStrictEqual`)/

Reply:
could you elaborate, I'd like to better understand your point. I use `.equalTo` for comparing simple types such as numbers. I don't think there is a difference to `.toStrictEqual()` when comparing numbers?